src/icons/build-icons/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "dependencies": {},
   "devDependencies": {
-    "grunt": ">=1.3.0",
+    "grunt": ">=1.5.2",
```

Comment:
```suggestion
    "grunt": "*",
```
So we don't need to care about its version anymore.

Reply:
Done, Thanks for the suggestion.